### PR TITLE
Add more types to parquet reader/writer

### PR DESCRIPTION
**Pull request by acquamarin**

### CODE REVIEW

File: src/common/types/timestamp_t.cpp [hunk 1]
```diff
@@ -256,12 +256,20 @@ void Timestamp::convert(timestamp_t timestamp, date_t& out_date, dtime_t& out_ti
...
-    return timestamp_t(epochSec * Interval::MICROS_PER_SEC);
+timestamp_t Timestamp::fromEpochMilliSeconds(int64_t ms) {
+    return fromEpochMicroSeconds(ms * Interval::MICROS_PER_MSEC);
+}
+
+timestamp_t Timestamp::fromEpochSeconds(int64_t sec) {
+    return fromEpochMicroSeconds(sec * Interval::MICROS_PER_SEC);
+}
+
+timestamp_t Timestamp::fromEpochNanoSeconds(int64_t ns) {
```
Comment (ray6080) [inline src/common/types/timestamp_t.cpp:271]: Is there any test covering this? `fromEpochNanoSeconds` If not, let's add one for it!

File: src/processor/operator/persistent/writer/parquet/column_writer.cpp [hunk 2]
```diff
@@ -50,6 +50,7 @@ std::unique_ptr<ColumnWriter> ColumnWriter::createWriterRecursive(
     }
     auto schemaIdx = schemas.size();
     switch (type->getLogicalTypeID()) {
+    case LogicalTypeID::UNION:
```
Comment (ray6080) [inline src/processor/operator/persistent/writer/parquet/column_writer.cpp:53]: I think it's better to separate this change to another PR for reading and writing UNION. so we can properly test it.

**Review by ray6080 — APPROVED** [review]

File: src/common/types/timestamp_t.cpp (hunk 1 above)
Comment (acquamarin) [reply src/common/types/timestamp_t.cpp:271]: Pyarrow doesn't automatically convert the nano seconds timestamp to micro seconds...
I can't find a tool that generates a nano second timestamp column
